tests/utils.go

```diff
@@ -348,10 +348,7 @@ func NewRandomVirtualMachineInstanceWithBlockDisk(imageUrl, namespace string, ac
 	if !libstorage.HasCDI() {
 		Skip("Skip DataVolume tests when CDI is not present")
 	}
-	sc, exists := libstorage.GetRWOBlockStorageClass()
```

Comment:
Keep this as is for now?

Reply:
why? its a small change, and the exact same code is used twice.

Reply:
Is it related to this PR?
I don't understand this change. To me it merits its own commit message explaining why it is good.

```
Remove NewRandomVirtualMachineInstanceWithDisk from datavolume test
- Remove NewRandomVirtualMachineInstanceWithDisk from datavolume test
- make use of GetRWOBlockStorageClass under NewRandomVirtualMachineInstanceWithBlockDisk
```

Reply:
Done